compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/matchpatterns/ErrorMessageMatchPatternNode.java

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.model.tree.matchpatterns;
+
+import org.ballerinalang.model.tree.Node;
+
+/**
+ * The interface with the APIs to implement the error-message-match-pattern.
+ *
+ * @since Swan Lake
+ */
+public interface ErrorMessageMatchPatternNode extends Node {
+    SimpleMatchPatternNode getSimpleMatchPattern();
```

Comment:
In this PR, some classes have a new line before the first line of the class/interface, while others don't have. Can we use a consistent formatting?

Reply:
Fixed